src/styles/utils/_normalize.scss

```
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  min-height: 100vh;
+  scroll-behavior: smooth;
+}
```

In `noramalize.scss` we dont't use class names only tag names.
```suggestion
body {
  min-height: 100vh;
  scroll-behavior: smooth;
}
```

---

I fixed almost everything